libs/speculos-transport/src/index.ts

```diff
@@ -181,6 +196,12 @@ export async function createSpeculosDevice(
           `${dependency}:./apps/${conventionalAppSubpath(model, firmware, dependency, appVersion)}`,
         ]
       : []),
+    ...(dependencies !== undefined
+      ? dependencies.flatMap(dependency => [
+          "-l",
+          `${dependency.name}:./apps/${conventionalAppSubpath(model, firmware, dependency.name, dependency.appVersion ? dependency.appVersion : "1.1.0")}`,
```

Comment:
why app version here is 1.1.0 and in common/speculos it's 1.1.1

Reply:
fallback value, I can put something else

Reply:
I understand that it's feedback value, the question is why it's different from what is used in another file, shouldn't they be the same?

Reply:
since it's a fallback value, there's not guarantee this version is available for selected `appVersion` 
will do anyway